### ISSUE TITLE
jquery 3.2.1 incompatibility

### ISSUE THREAD

**Issue by erasmodanarni** [yourlabs/django-autocomplete-light#928]
The latest version of DAL is incompatible with:

`<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>`

Thanks!

**Comment by jpic** [comment]
Thanks ! pull request welcome ;)

**Comment by etoulas** [comment]
I probably came across the same issue. For me it was the `autocomplete.ModelSelect2` that didn't work: the browser just showed a dropdown with one line containing only "---------------".
It's difficult to tell what exactly is going wrong as I don't get any errors in the console.

I included the `jquery-3.2.1.slim.min.js` following the Bootstrap 4(!) starter template. Simply replacing the jQuery version with https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js (from the Bootstrap 3 starter template) help to get the `ModelSelect2` working.

However, I'm not sure how much this will break Bootstrap 4...
I tried to hunt down some minimum jQuery version requirements for both bootstrap4 as well as django-autocomplete-light but I couldn't figure out what would be the common jQuery version plays well with both projects.
Any clues @jpic ?

**Comment by etoulas** [comment]
I noticed the incompatibility when using `autocomplete.ModelSelect2`: the ajax request didn't fire. I got this error message in the console:
```
TypeError: $.ajax is not a function: select2.full.js:3427:24
transport: http://127.0.0.1:8000/static/autocomplete_light/vendor/select2/dist/js/select2.full.js:3427:24
```
It seems that select2 requires jQuery 2.1.0 (see https://github.com/select2/select2/blob/c155ca5b60c2769da333d3922f4a14f54e9e50e4/vendor/jquery-2.1.0.js)

I included the file `jquery-3.2.1.slim.min.js` in my Django template following the Bootstrap 4(!) starter template. The ajax worked when I replaced it with version 2.1.0.

Maybe there is a way to have select2 to use it's own jQuery version and ignore the one that bootstrap4 requires. That would be great.

**Comment by etoulas** [comment]
I found the issue with the $.ajax.
I was using the "slim" version which does not include ajax functionality according to this SO: https://stackoverflow.com/questions/44212202/my-javascript-is-returning-this-error-ajax-is-not-a-function

Use `jquery-3.2.1.min.js` instead of slim `jquery-3.2.1.slim.min.js`.
